README.rst

```diff
@@ -1,48 +1,49 @@
-A package for computing solar differential emission measure distributions
--------------------------------------------------------------------------
+Sunkit-Dem 
+==========
 
-.. image:: http://img.shields.io/badge/powered%20by-SunPy-orange.svg?style=flat
+|SunPy Badge| |matrix|
+
+.. |SunPy Badge| image:: http://img.shields.io/badge/powered%20by-SunPy-orange.svg?style=flat
     :target: http://www.sunpy.org
     :alt: Powered by SunPy Badge
+.. |matrix| image:: https://img.shields.io/matrix/sunpy:openastronomy.org.svg?colorB=%23FE7900&label=Chat&logo=matrix&server_fqdn=openastronomy.modular.im
+   :target: https://riot.im/app/#/room/#sunkit-dem:openastronomy.org
 
-License
```

Comment:
probably shouldn't be removing the license?

Reply:
I think it's just been moved below

Reply:
Ah yes, apologies.